### ISSUE TITLE
Ability to expand responsive toggles at once

### ISSUE THREAD

**Issue by AchalJ** [beaverbuilder/feature-requests#296]
Currently, we have to click 3 times to put change/add value for small device breakpoint as Default > Large > Medium

Now to go back to the medium device breakpoint again 3 clicks are needed Small > Default > Large

The worst situation is when you have resource-heavy content on the page like videos, or sliders slow down the builder when switching from each responsive preview.

It would be great to have this toggle just reveal all at once with a single click to make it easy to switch between them. Please see the screenshot below:

![image](https://user-images.githubusercontent.com/12668839/209083054-ac5d9f0a-35c5-4503-b865-b9ef60c5960a.png)

**Comment by jodzeee** [comment]
Yes, please! The next version has improvements in responsive editing hopefully this is one of them. @AchalJ I heard the pre-release is out if you want to test it.

**Comment by AchalJ** [comment]
@jodzeee Indeed, there are loads of improvements in BB 2.7 pre regarding responsive preview but it just lacks the feature I suggested above. Hope the BB team also implements this in 2.7. It would make it very convenient and we don't have to go to each responsive preview every time. :)
